crawl-ref/source/cloud-type.h

```diff
@@ -42,6 +42,7 @@ enum cloud_type
     CLOUD_EMBERS,
 #endif
     CLOUD_FLAME,
+    CLOUD_ELECTRICITY,
```

Comment:
Inserting an enum into the middle of a list of enums will break save compatibility if the enum is marshalled (included in save files), as this one is. Anyone who loads up an old save after this change will find all degeneration clouds have turned into electricity clouds, and all blastsparks turned into degeneration.

You need to insert this at the end of the enum, right before NUM_CLOUD_TYPES.